docs/concepts/flow-runners.md

```diff
@@ -200,13 +200,15 @@ The Prefect CLI command `prefect orion kubernetes-manifest` automatically genera
 
 `KubernetesFlowRunner` supports the following settings:
 
-| Attributes | Description |
-| ---- | ---- |
-| image | String specifying the tag of a Docker image to use for the Job. |
-| namespace | String signifying the Kubernetes namespace to use. |
-| labels | Dictionary of labels to add to the Job. |
-| image_pull_policy | The Kubernetes image pull policy to use for Job containers. |
-| restart_policy | The Kubernetes restart policy to use for Jobs. |
-| stream_output | Bool indicating whether to stream output from the subprocess to local standard output. |
+| Attributes | Description                                                                                                            |
+| ---- |------------------------------------------------------------------------------------------------------------------------|
+| image | String specifying the tag of a Docker image to use for the Job.                                                        |
+| namespace | String signifying the Kubernetes namespace to use.                                                                     |
+| labels | Dictionary of labels to add to the Job.                                                                                |
+| image_pull_policy | The Kubernetes image pull policy to use for Job containers.                                                            |
+| restart_policy | The Kubernetes restart policy to use for Jobs.                                                                         |
+| stream_output | Bool indicating whether to stream output from the subprocess to local standard output.                                 |
+| node_selector | [Kubernetes node_selector documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector)  |
```

Comment:
```suggestion
| node_selector | [Kubernetes node_selector documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector).  |
```

Reply:
Could this be written in a way that indicates the format of the data here, following the example of the other descriptions? Link to the docs is good. Please use full sentence capitalization and punctuation.